net.gpro.gproapp.yml

```diff
@@ -0,0 +1,44 @@
+app-id: net.gpro.gproapp
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: startgpro
+rename-icon: gpro
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
Doesn't work on wayland

```
assing the following arguments to Electron: --ozone-platform-hint=auto
[2:1128/072955.981543:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
runCount2
stesmworks,  {
  init: [Function (anonymous)],
  restartAppIfNecessary: [Function (anonymous)],
  electronEnableSteamOverlay: [Function (anonymous)],
  SteamCallback: {
    PersonaStateChange: 0,
    SteamServersConnected: 1,
    SteamServersDisconnected: 2,
    SteamServerConnectFailure: 3,
    LobbyDataUpdate: 4,
    LobbyChatUpdate: 5,
    P2PSessionRequest: 6,
    P2PSessionConnectFail: 7
  }
}
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
dlopen failed trying to load:
/home/test/.steam/sdk64/steamclient.so
with error:
/home/test/.steam/sdk64/steamclient.so: cannot open shared object file: No such file or directory
[S_API] SteamAPI_Init(): Sys_LoadModule failed to load: /home/test/.steam/sdk64/steamclient.so
[2:1128/072956.235500:ERROR:wayland_connection.cc(198)] Failed to connect to Wayland display: No such file or directory (2)
[2:1128/072956.235524:ERROR:ozone_platform_wayland.cc(264)] Failed to initialize Wayland platform
[2:1128/072956.235528:ERROR:env.cc(258)] The platform failed to initialize.  Exiting.
```

Reply:
If you want to experiment with `--ozone-platform-hint=auto` use fallback-x11+wayland sockets, otherwise don't pass that and follow the Electron default (keep x11 only)